source/text/Format.cpp

```diff
@@ -195,6 +195,10 @@ string Format::Number(double value)
 {
 	if(!value)
 		return "0";
+	else if(std::isnan(value))
+		return "???";
```

Comment:
why not NaN?

Reply:
Because it's more consistent with other "unknown" values in the game. This would mainly be used in the UI, and NaN is not necessarily something people can interpret. ~~Also, I have no idea what would happen if a ship with a nan value was actually flying around - probably `???` represents the behaviour better than `NaN`.~~